renovate.json

```
@@ -0,0 +1,4 @@
+{
+  "extends": ["github>KinsonDigital/.github//config/renovate-config.json"]
```

Update the path section.
You forgot the _**.github**_ folder as part of your path.

```suggestion
  "extends": ["github>KinsonDigital/.github//.github/config/renovate-config.json"]
```